src/cljc/rems/common/roles.cljc

```diff
@@ -32,7 +33,29 @@
             (boolean (some (set roles) context/*roles*)))
      :cljs (some (set roles) (:roles @(rf/subscribe [:identity])))))
 
+(defn is-with-organization?
+  "Determines if the user is part of an organization based on the passed in ID."
+  [organization-id]
+  #?(:cljs (let [owned-organizations @(rf/subscribe [:owned-organizations])]
+             (->> owned-organizations
+                  (map #(get-in % [:organization/id]))
+                  (in? organization-id)))))
+
+(defn has-permission?
```

Comment:
Again here we could say something like `has-ownership-of?`. I believe we don't need to necessarily pass here other roles for our case (i.e., we don't need to both check for `:organization-owner` role and membership in organization owners as the former is calculated from the latter and the latter is therefore enough). Also isn't it `or` so either you are the (super) `owner` or the owner of the organization? So you don't need to be both.